File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ActorType.java
package ru.iqchannels.sdk.schema;

import com.google.gson.annotations.SerializedName;

/**
 * Copyright iqstore.ru. All Rights Reserved.
 */
public enum ActorType {
    @SerializedName("")ANONYMOUS,
    @SerializedName("client")CLIENT,
    @SerializedName("user")USER,
    @SerializedName("system")SYSTEM
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ActorType.kt
package ru.iqchannels.sdk.schema

import com.google.gson.annotations.SerializedName

enum class ActorType {
	@SerializedName("")
	ANONYMOUS,
	@SerializedName("client")
	CLIENT,
	@SerializedName("user")
	USER,
	@SerializedName("system")
	SYSTEM
}
